apps/wpcom-block-editor/src/wpcom/features/override-edit-site-back-button.js

```diff
@@ -22,13 +21,16 @@ function overrideBackToDashboardButton() {
 
 	registerPlugin( 'a8c-wpcom-block-editor-site-editor-back-to-dashboard-override', {
 		render: function SiteEditorCloseFill() {
-			return createElement( SiteEditorDashboardFill, null, [
-				createElement( NavigationBackButton, {
-					backButtonLabel: __( 'Dashboard' ),
-					className: 'edit-site-navigation-panel__back-to-dashboard',
-					href: `https://wordpress.com/home/${ siteSlug }`,
-				} ),
-			] );
+			return (
+				<SiteEditorDashboardFill>
+					<NavigationBackButton
+						backButtonLabel={ __( 'Dashboard' ) }
+						// eslint-disable-next-line wpcalypso/jsx-classname-namespace
```

Comment:
Eslint wants us to prefix the classname to with the `overrideBackToDashboardButton` function name. We, however, only want to override the back to admin dashboard href. Any related styles should stay the same. Because of this, we keep originally assigned classname in Gutenberg.